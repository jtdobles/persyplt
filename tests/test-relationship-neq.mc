void main(){
	float a = 3.0;
	if(a != 2.0){
		prints('a is 2.0');
	}else{
		prints('a is not 2.0');
	}
	if(a != 3.0){
		prints('a is 3.0');
	}else{
		prints('a is not 3.0');
	}
}