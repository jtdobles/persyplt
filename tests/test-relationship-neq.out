'a is 2.0'
'a is not 3.0'
